frontend/sozisel-app/src/components/PresenterSession/EventsTimline/LiveEventDetails/PollLiveEventDetails/PollLiveEventDetails.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement, useContext } from "react";
+import { LiveEventContext } from "../LiveEventDetails";
+import PollPieChart from "../../../../utils/PollPieChart/PollPieChart";
+import TextSection from "../TextSection";
+import Timer from "../Timer";
+import useFetchEventLiveResult from "../../../../../hooks/useFetchEventLiveResult/useFetchEventLiveResult";
+import { useTranslation } from "react-i18next";
+
+const PollLiveEventDetails = (): ReactElement => {
+  const { t } = useTranslation("common");
+  const {
+    activeEvent,
+    event,
+    sessionId,
+    onFinishCallback,
+    participantsNumber,
+  } = useContext(LiveEventContext);
+
+  const { mostAnsweredOption, totalVoters, optionsSummary } =
+    useFetchEventLiveResult(sessionId, event, activeEvent.id, "PollResult");
+
+  if (event.eventData.__typename !== "Poll") {
+    return <></>;
```

Comment:
Is this necessary? You are already checking typename in `LiveEventDetails.tsx`.

Reply:
I agree, if this if statements happens to trigger that would mean there is something wrong. This case will just silence it which we probably don't want.

Reply:
Yes, it is useless to perform this check here.